tools/client-plugins/browser-scripts/typescript-worker.ts

```diff
@@ -39,18 +39,22 @@ interface CancelEvent extends MessageEvent {
   };
 }
 
-const TS_VERSION = '5'; // hardcoding for now, in the future this may be dynamic
+const TS_VERSION = '5.7.3'; // hardcoding for now, in the future this may be dynamic. Must be fully specified version
```

Comment:
@freeCodeCamp/dev-team has the future arrived yet 😄? I can see this becoming tech-debt.

Reply:
https://github.com/freeCodeCamp/freeCodeCamp/pull/59291#discussion_r2000442352 Is this related?